yt_dlp/extractor/youtube.py

```diff
@@ -336,14 +336,20 @@ def _extract_identity_token(self, webpage, item_id):
 
     @staticmethod
     def _extract_account_syncid(data):
-        """Extract syncId required to download private playlists of secondary channels"""
-        sync_ids = (
-            try_get(data, lambda x: x['responseContext']['mainAppWebResponseContext']['datasyncId'], compat_str)
-            or '').split("||")
+        """
+        Extract syncId required to download private playlists of secondary channels
+        params:
+        - data: can be either response or ytcfg
```

Comment:
```suggestion
        @param data Either response or ytcfg
```

This is the standard, although I probably missed this in the earlier PRs

Reply:
Ah alright. thanks